### PR TITLE
Updated Readme to be more readable and contain more information.

### PR DESCRIPTION
**Pull request by JSpenced**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,74 @@
...
-ice_recur 1.5.2  4 Dec 2018: Added -f
+You put your entries in `$TODO_DIR/ice_recur.txt` , and add something like this:
+
+      ~/bin/todo.sh ice_recur
+
+to your crontab, to run once a day.
+
+Every entry that matches the current day or previous days after `.ice_recur_completed` was last touched will be added, as long as there is no other entry with the same text content.
+
+Comments can be added to the file using `#` but `ice_recur.txt` can contain no blank lines between entries or the script won't execute properly.
```
Comment (JSpenced) [inline README.md]: @rlpowell This is true on my computer, so I am not sure if it was meant to be like this.

Comment (rlpowell) [reply README.md]: Mine definitely has blank lines and they're no trouble.  Can you tell me exactly what happens to you when you have blank lines?

**Comment by rlpowell** [comment]
I am really not interested in maintaining the options help in two different places.

**Comment by rlpowell** [comment]
The rest of it looks great, though.

File: README.md (hunk 1 above)
Comment (JSpenced) [reply README.md]: Sorry, this was a mistake. I know I had this problem in an old version but didn't test in the current version. It appears to be fixed.

**Comment by JSpenced** [comment]
Okay, I removed the options and updated the installation section a bit more when I removed the issue about comments since it was wrong.

**Pull request merged**